templates/players.html

```diff
@@ -12,9 +12,9 @@ <h1 class='text-center text-muted mb-3'>Players</h1>
   {% for row in player_list %}
   <tr>
     <td>{{row.name}}</td>
-    <td>{{row.name.lower() == "nadir"}}</td>
+    <td>{{row.name.lower() == "nadir" || row.name.lower() == "adonis" || row.name.lower() == "elia" || row.name.lower() == "nicolas g"}}</td>
```

Comment:
I think there's a typo:
```suggestion
    <td>{{row.name.lower() == "nadir" || row.name.lower() == "adonis" || row.name.lower() == "nicolas g"}}</td>
```

Reply:
No no it's all good! 👍 Merging